src/foundry/client/data/documents/actor.d.mts

```diff
@@ -508,55 +509,46 @@ declare global {
       | Document.OnDeleteDescendantDocumentsArgs<Actor.Stored, Actor.DirectDescendant, Actor.Metadata.Embedded>
       | Item.OnDeleteDescendantDocumentsArgs;
 
-    /**
-     * @deprecated {@link Actor.Database | `Actor.DatabaseOperation`}
-     */
-    // eslint-disable-next-line @typescript-eslint/no-deprecated
-    interface DatabaseOperations extends Document.Database.Operations<Actor.Implementation> {}
-
-    /**
-     * @deprecated {@link Actor.SubType | `Actor.SubType`}
-     */
-    type TypeNames = Actor.SubType;
-
-    /**
-     * @deprecated {@link Actor.CreateData | `Actor.CreateData`}
-     */
-    interface ConstructorData extends Actor.CreateData {}
-
-    /**
-     * @deprecated {@link Actor.implementation | `Actor.ImplementationClass`}
-     */
-    type ConfiguredClass = ImplementationClass;
+    interface GetDefaultArtworkReturn {
+      img: string;
+      texture: GetDefaultArtworkTextureReturn;
+    }
 
-    /**
-     * @deprecated {@link Actor.Implementation | `Actor.Implementation`}
-     */
-    type ConfiguredInstance = Implementation;
+    interface GetDefaultArtworkTextureReturn {
+      src: string;
+    }
 
     type ItemTypes = {
       [SubType in Game.Model.TypeNames<"Item">]: Array<Item.OfType<SubType>>;
```

Comment:
I think this can just be `Item.SubType`, no? I think this predates `Item.SubType` but is equivalent. Double check for me though.